code/datums/sexcon/sexcon.dm

```diff
@@ -94,10 +94,13 @@
 	if(!user.defiant && !victim.defiant)
 		return FALSE
 	// Need to violate AFK clients
-	if(victim.mind && victim.mind.key && !victim.client)
+	if(!victim.mind || !victim.mind.key || !victim.client) // Changed to OR statements to remove ZAPE of mobs without minds or keys
```

Comment:
Nah, imo- I think this is fine, we should only check for defiantness, which is stored on the mob, so this should be fine.

Reply:
Honestly let's jsut do `!victim.mind?.key`, that should be null if mind is null and is more succinct.